lib/fcrepo_wrapper.rb

```diff
@@ -22,7 +22,7 @@ def self.default_instance_options=(options)
   end
 
   def self.default_instance(options = {})
-    @default_instance ||= FcrepoWrapper::Instance.new default_instance_options.merge(options)
```

Comment:
Not quite, the default port is set through default_instance options in [FcrepoWrapper::Configuration.port](https://github.com/cbeer/fcrepo_wrapper/blob/master/lib/fcrepo_wrapper/configuration.rb#L98)
```
    def port
      # Check if the port option has been explicitly set to nil.
      # this means to start fcrepo_wrapper on a random open port
      return nil if options.key?(:port) && !options[:port]
      options[:port] || FcrepoWrapper.default_instance_options[:port]
    end
```
You could factor that out pretty easily and do something like the [.version](https://github.com/cbeer/fcrepo_wrapper/blob/master/lib/fcrepo_wrapper/configuration.rb#L41-L43) default instead:
```
    def version
      @version ||= options.fetch(:version, FcrepoWrapper.default_fcrepo_version)
    end
```

It looks like @cam156 had some reason for [Adding a setter for the default_instance_options](https://github.com/cbeer/solr_wrapper/commit/48ed28f042db253bf1da6889fe46f0e34f570818) in solr_wrapper and it got copied over here.

I'm not sure what you'd ever want to do that, and it took me a fair bit of time to wrap my head around what all was happening, so I'd be happy to take default_instance_options out and follow the pattern for the version defaulting instead.